subprojects/testfx-junit5/testfx-junit5.gradle

```diff
@@ -138,9 +139,11 @@ afterEvaluate {
                 options.compilerArgs = [
                         '--module-path', classpath.asPath,
                         '--add-modules', 'org.junit.jupiter.api',
+                        '--add-modules', 'org.junit.platform.commons',
```

Comment:
Let's remove these now that we have made this PR not change logging stuff.

Reply:
addressed by 75981dd